app/src/behaviors/behavior_rgb_underglow.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2020 Nick Winans <nick@winans.codes>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_rgb_underglow
+
+#include <device.h>
+#include <drivers/behavior.h>
+#include <logging/log.h>
+
+#include <dt-bindings/zmk/rgb.h>
+#include <zmk/rgb_underglow.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+struct behavior_rgb_underglow_config { };
+struct behavior_rgb_underglow_data { };
+
+static int behavior_rgb_underglow_init(struct device *dev)
+{
+  return 0;
+}
+
+static int on_keymap_binding_pressed(struct device *dev, u32_t position, u32_t action, u32_t _)
+{
+  switch (action)
+  {
+    case RGB_TOG:
+      zmk_rgb_underglow_toggle();
+      break;
```

Comment:
Instead of all the breaks, etc. since the action is really the *only* think you want to do, you might consider refactoring all of these to:

```
case RGB_TOG:
  return zmk_rb_underglow_toggle();
case BLAH:
```

And then the very end of the function can just `return -ENOTSUP;` if we didn't match a supported action.

This requires each of those called functions return `int` for success/errno, which is probably good anyways.